packages/eslint-plugin/test/__snapshots__/plugin.test.ts.snap

```diff
@@ -0,0 +1,655 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`plugin should have the expected exports 1`] = `
+{
+  "configs": {
```

Comment:
This is a new snapshot I added to the PR to baseline our exported plugin; you'll see in the commit that filtered out `npm-naming` that it was removed (but the rule remained as a rule).